src/components/GenericServerSideEdit/README.md

```diff
@@ -0,0 +1,90 @@
+# GenericServerSideEdit
+
+The `GenericServerSideEdit` component is a generic block edit component that uses server side rendering to display content.
+
+## Usage
+
+For a minimum working setup, all you need to do is import the `GenericServerSideEdit` component and assign it as the `edit` parameter when registering a block:
+
+```js
+import { registerBlockType } from '@wordpress/blocks';
+
+import blockMetadata from './block.json';
+import GenericServerSideEdit from '@humanmade/block-editor-components';
+
+registerBlockType( blockMetadata, {
+	edit: GenericServerSideEdit,
+} );
+
+```
+
+Optionally, you can use it inside an edit component with inspector controls:
+
+```js
+import { Panel, PanelBody } from '@wordpress/components';
+import { InspectorControls } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+import GenericServerSideEdit from '@humanmade/block-editor-components';
+
+/**
+ * Render the editor interface.
+ *
+ * @returns {import('react').ReactNode} Rendered editorial UI.
+ */
+export default function Edit( { attributes, context, setAttributes } ) {
+	return <GenericServerSideEdit
+		name="my/blockname"
+		attributes={ attributes }
+		context={ context }
+		inspectorControls={
```

Comment:
One thought on this approach, just wondering if its better to pass `inspectorControls` as a prop, or just allow using children? Thinking that if you wanted to use the `BlockControls` component for example, it wouldn't make sense to pass it here (although technically it would work)

```
<GenericServerSideEdit ...props>
    <BlockControls>
        ...
    </BlockControls>
    <InspectorControls>
        ...
    </InspectorControls>
</GenericServerSideEdit>
```

Reply:
hmmm possibly, but I'm mindful that there has been some limited success with server rendered blocks that have children, doing this would preclude that opportunity, while still requiring an edit component that could host a fragment. Doing `<> <GenericServerSideEdit ...props /> <InspectorControls>......</InspectorControls></>` would functionally separate this component from the problem in a way that future proofs it.

Part of the issue the props was trying to address was how to add inputs and other controls for attributes, but if we're doing this the need for the edit component to wrap around this makes that somewhat defunct as `useState`or `useEffect` etc calls will be needed

Reply:
OK that's neat. Seems like a good reason not to use children to me 👍